src/lib.rs

```diff
@@ -366,20 +365,17 @@ fn from_der_(i: &[u8], start_offset: usize) -> Result<Vec<ASN1Block>, ASN1Decode
         if class != ASN1Class::Universal {
             if constructed {
                 // Try to read as explicitly tagged
-                match from_der_(body, start_offset + index) {
-                    Ok(mut items) => {
-                        if items.len() == 1 {
-                            result.push(ASN1Block::Explicit(
-                                class,
-                                soff,
-                                tag,
-                                Box::new(items.remove(0)),
-                            ));
-                            index += len;
-                            continue;
-                        }
+                if let Ok(mut items) =from_der_(body, start_offset + index) {
```

Comment:
I feel a bit sorry for marking such a little thing, but there is a space missing after the `=` ;)

Nice find of the time parsing panic btw.

Reply:
good catch; I've pushed and updated version with the space you asked for :)